sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -10,12 +10,16 @@
 - Added property `details` on returned poller objects which contain long-running operation metadata.
 - Added `TextAnalysisLROPoller` and `AsyncTextAnalysisLROPoller` protocols to describe the return types from long-running operations.
 - Added `cancel` method on the poller objects. Call it to cancel a long-running operation that's in progress.
+- Added property `kind` to `RecognizeEntitiesResult`, `RecognizePiiEntitiesResult`, `AnalyzeHealthcareEntitiesResult`,
```

Comment:
Do you think this is something we should do across languages for much the same reason? I would say .NET is pretty inconsistent with type checks vs. discriminator checks, but wouldn't be opposed to it for each way.

Reply:
Discussed offline, but basically having a `kind` property makes more sense for Python/JS designs since we return a heterogenous list of result | error.